classes/class-ep-debug-bar-query-output.php

```diff
@@ -24,13 +24,19 @@ class EP_Debug_Bar_Query_Output {
 	 * @return void
 	 */
 	public static function render_query( $query, $type = '' ) {
-
-		$query_time = ( ! empty( $query['time_start'] ) && ! empty( $query['time_finish'] ) ) ? $query['time_finish'] - $query['time_start'] : false;
-
-		$result   = wp_remote_retrieve_body( $query['request'] );
-		$response = wp_remote_retrieve_response_code( $query['request'] );
-
-		$class = $response < 200 || $response >= 300 ? 'ep-query-failed' : '';
+		$error         = '';
+		$query_time    = ( ! empty( $query['time_start'] ) && ! empty( $query['time_finish'] ) ) ? $query['time_finish'] - $query['time_start'] : false;
+		$result        = wp_remote_retrieve_body( $query['request'] );
+		$response      = wp_remote_retrieve_response_code( $query['request'] );
+		$class         = $response < 200 || $response >= 300 ? 'ep-query-failed' : '';
+		$log['result'] = json_decode( $result, true );
+
+		if ( class_exists( 'ElasticPress\StatusReport\FailedQueries' ) && class_exists( 'ElasticPress\QueryLogger' ) ) {
+			$query_logger   = apply_filters( 'ep_query_logger', new \ElasticPress\QueryLogger() );
+			$failed_queries = new ElasticPress\StatusReport\FailedQueries( $query_logger );
```

Comment:
@MARQAS one possible usage of the `ep_query_logger` filter is that a dev could make it empty, completely disabling the query logging functionality. [In the EP plugin](https://github.com/10up/ElasticPress/blob/develop/includes/classes/Screen/StatusReport.php#L88), we have:
```php
		$query_logger = apply_filters( 'ep_query_logger', new \ElasticPress\QueryLogger() );
		if ( $query_logger ) {
			$reports['failed-queries'] = new \ElasticPress\StatusReport\FailedQueries( $query_logger );
		}
```
We will need to have the same check here as well.